### ISSUE TITLE
Port Billing Re-Invoice pages

### ISSUE THREAD

**Issue by JimDalton** [TransNexus/NexOSS#172]
Page Paths in Legacy UI

Add Request:    /billing/oss_billing_add_invoice_request.htm
View Pending Requests:  /billing/oss_billing_pending_reinvoice_q.htm
View Processed Requests:  /billing/oss_billing_processed_reinvoice_q.htm

**Comment by DmitryIsakbayev** [comment]
@fenichelar,

The "Add Request" to re-invoice is similar to "Add Request" to export rate plan.  I am using the paths.exportRatePlan.  We need either paths.requestToReInvoice or make .exportRatePlan generic so that we can use for both use cases.

**Comment by DmitryIsakbayev** [comment]
Status: "Add Request" - waiting for changes.  The other two pages have been ported.

**Comment by fenichelar** [comment]
@DmitryIsakbayev Should any data be attached with the `requestToReInvoice` request.

**Comment by DmitryIsakbayev** [comment]
@fenichelar, yes.  Just like the `.exportRatePlan`, it should data for the selected row.

**Comment by fenichelar** [comment]
@DmitryIsakbayev Done. Please test. `paths.requestToReinvoice`

**Comment by fenichelar** [comment]
@DmitryIsakbayev Just so you know how I set it up:

`executeGetAction` and ` executePostAction` exist on sails. Get does not attach data, post attaches data. A get or post request is made to this url by Ember: `/api/pages/`**PAGENAME**`/actions/`**ACTIONNAME**. Action name defines the path parameter to use like `exportRatePlan`.

Adding a new action requires no changes to sails. A new button needs to be created on Ember. The button is needed because buttons have variables that keep track of the current state of the action. This makes buttons display "Loading" while the action is being executed. Icons and colors are also defined in Ember. So the actions are generic on Sails but not on Ember.

**Comment by DmitryIsakbayev** [comment]
Switched to requestToReinvoice.  Please test.
